### ISSUE TITLE
issue of support of ”nn.BatchNorm1d/nn.BatchNorm2d“ (AIS-2177)

### ISSUE THREAD

**Issue by GeorgeBohw** [espressif/esp-dl#269]
### Checklist

- [x] Checked the issue tracker for similar issues to ensure this is not a duplicate.
- [x] Provided a clear description of your suggestion.
- [x] Included any relevant context or examples.

### Issue or Suggestion Description

请问，espdl(esp32s3)支持”nn.BatchNorm1d/nn.BatchNorm2d“么？官方文档显示不支持这个层，但是我的神经网络中有这个层，模型在板子上运行没有报错成功运行了，这个是怎么回事，与官方显示不支持这个层就矛盾了，是转为cpu执行了么？

**Comment by sun-xiangyu** [comment]
> ### Checklist
> * [x]  Checked the issue tracker for similar issues to ensure this is not a duplicate.[x]  Provided a clear description of your suggestion.[x]  Included any relevant context or examples.
> 
> ### Issue or Suggestion Description
> 请问，espdl(esp32s3)支持”nn.BatchNorm1d/nn.BatchNorm2d“么？官方文档显示不支持这个层，但是我的神经网络中有这个层，模型在板子上运行没有报错成功运行了，这个是怎么回事，与官方显示不支持这个层就矛盾了，是转为cpu执行了么？

这个是因为在通过esp-ppq量化和转化的过程中，conv + batchnorm 被自动融合成了一个新的conv 算子， 这样可以有效提高模型的推理效率。

**Comment by GeorgeBohw** [comment]
@sun-xiangyu 是不是可以这样理解：某个layer只要在板子上可以正确运行，就可以理解为是被支持的layer，不被支持的layer在板子上运行是会报错的。这么理解对么？

**Comment by sun-xiangyu** [comment]
是的，不支持的算子肯定会报错的。有些我们没有列出来但是可以正常运行的算子，大概率是在模型转化的过程中被优化掉了。

**Comment by GeorgeBohw** [comment]
@sun-xiangyu 非常感谢！
